foundation-extensions/mobile-core/lifecycle/README.md

```diff
@@ -20,16 +20,25 @@ Sessions contain information about the app's current lifecycle, such as the devi
 {% endtab %}
 
 {% tab title="iOS (AEP 3.x)" %}
+
+Add the AEPLifecycle extension and it's dependency, the [Mobile Core](https://aep-sdks.gitbook.io/docs/using-mobile-extensions/mobile-core) extension to your project using Cocoapods.
```

Comment:
"... and its dependency, the Mobile Core extension, to your project..."

Reply:
Additionally, the links here should be relative links within the repo structure, rather than absolute links that link directly to a URL. (This isn't a necessary change to make now, but would be nice to have fixed.)

Reply:
@calebk1 Thanks! Let me know if that works.